src/tasks/tasks.py

```diff
@@ -25,13 +25,13 @@
 
```

Comment:
It looks like there are some changes in here that aren't related to CoLA. Either get rid of those or explain why they have to be here.

Reply:
create_subset_scorer, update_subset_scorers, collect_subset_scores are mine. I put them outside cola because they can be reused for other tasks that need find-grained metrics. 
Any other changes outside cola are from load_tsv branch.